examples/datepicker.js

```diff
@@ -4,6 +4,7 @@
 const exports = {};
 
 import './datepicker.css';
+import 'jquery-ui/themes/all.css';
```

Comment:
Why do you take all.js here?
Shouldn't the CSS be imported in the source code needing it?